bin/upgrading/en.dat

```diff
@@ -3983,7 +3983,7 @@ taglib.error.access=You are not allowed to tag entries in this journal.
 
 taglib.error.add=You are not allowed to create new tags for this journal; the entry was not tagged with [[tags]].
 
-taglib.error.delete=You are not allowed to delete tags from entries for this journal; the entry is still tagged with [[tags]].
+taglib.error.delete2=You are not allowed to delete tags from entries for this journal; the entry is still tagged with [[tags]].  Any tags you tried to add have not been added.
```

Comment:
I think this information would be more easily parsed if the list of tags continued to be at the end of the line.  So something like: "You are not allowed to delete tags from entries for this journal; any tags you tried to add have not been added. The entry is still tagged with [[tags]]."

Might also be better to break it into two lines at this point.